src/main/jbake/content/blog/2016/09/06/consuming-hawkular-api-over-ssl.adoc

```diff
@@ -0,0 +1,82 @@
+= Consuming Hawkular API over SSL with self signed certificates
+Josejulio Martínez
+2016-9-6
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+In this post I'll show you how to trust self-signed certificates for use with Ruby client, used on projects such as ManageIQ and HawkFX.
+
+If you haven't already, follow this guide for
+link:http://www.hawkular.org/hawkular-services/docs/installation-guide/secure-comm.html[securing communications]
+ on your Hawkular server, because I'll make use of the files generated by that guide.
```

Comment:
because "we'll make use..." instead of "I'll use", so that users feel like they are doing it with you :)